cobalt/webdriver/util/dispatch_command_factory.h

```diff
@@ -92,13 +92,13 @@ std::unique_ptr<base::Value> ToValue(const base::Optional<T>& value) {
 
 // Template specialization for std::string.
 template <>
-std::unique_ptr<base::Value> ToValue(const std::string& value) {
+inline std::unique_ptr<base::Value> ToValue(const std::string& value) {
```

Comment:
Why is inline needed?

Reply:
Because i'ts a fully specialized template, it's really a function, not a template anymore. Leaving those in the headers causes duplicate symbols and is effectively a violation of one definition rule. 

The fully clean way to do this is to leave the declaration in the header and move the implementation into a CPP file - but there's no practical difference in this case because they all only have one call site in production code, inlining has the same effecti.